tests/e2e/steps_permissionless_ics.go

```diff
@@ -0,0 +1,213 @@
+package main
+
+import (
+	"time"
+
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	e2e "github.com/cosmos/interchain-security/v6/tests/e2e/testlib"
+)
+
+// stepsPermissionlessICS tests
+// - starting multiple permissionless consumer chains with the same chain ID
+// - opt-in of a validator on two different chains with the same chain ID
+// - taking ownership of a consumer chain
+func stepsPermissionlessICS() []Step {
+	s := concatSteps(
+		[]Step{
+			// Start the provider chain
+			{
+				Action: StartChainAction{
+					Chain: ChainID("provi"),
+					Validators: []StartChainValidator{
+						{Id: ValidatorID("alice"), Stake: 100000000, Allocation: 10000000000},
+						{Id: ValidatorID("bob"), Stake: 200000000, Allocation: 10000000000},
+						{Id: ValidatorID("carol"), Stake: 300000000, Allocation: 10000000000},
+					},
+				},
+				State: State{
+					ChainID("provi"): ChainState{
+						ValPowers: &map[ValidatorID]uint{
+							ValidatorID("alice"): 100,
+							ValidatorID("bob"):   200,
+							ValidatorID("carol"): 300,
+						},
+					},
+				},
+			},
+
+			// Initialize a permissionless chain with ChainID `consu`
+			// - create the consumer chain
+			// - opt-in a validator
+			// - launch the chain
+			// - TODO (PERMISSIONLESS): 'start the chain'. Note: due to hermes we can't yet start two chains with the same ChainID as it's not yet supported
+
+			{
+				Action: CreateConsumerChainAction{
+					Chain:         ChainID("provi"),
+					From:          ValidatorID("alice"),
+					ConsumerChain: ChainID("cons2"), // test chain "cons2" is configured with ChainID "consu"
+					SpawnTime:     uint(time.Minute * 3),
+					InitialHeight: clienttypes.Height{RevisionNumber: 0, RevisionHeight: 1},
+					TopN:          0,
+				},
+				State: State{},
+			},
+			{
+				Action: OptInAction{
+					Chain:     ChainID("cons2"),
```

Comment:
Having this named `ChainID` while the underlying chain is is `consu` could be hard to parse. I wonder if we could rename this to something else, and introduce `ConsumerId ConsumerID` in all the actions instead of using `ChainID`.

Reply:
ConsumerID would also be misleading as it is not possible at that stage to have a match with the consumer ID we get when creating the consumer chain. But I do agree to come up with something which helps to differntiate this.
Added this to #2198